tests/Phinx/Config/ConfigTest.php

```diff
@@ -13,32 +13,6 @@
  */
 class ConfigTest extends AbstractConfigTest
 {
-    /**
-     * @covers \Phinx\Config\Config::__construct
-     * @covers \Phinx\Config\Config::getConfigFilePath
-     */
-    public function testConstructEmptyArguments()
-    {
-        $config = new Config([]);
-        // this option is set to its default value when not being passed in the constructor, so we can ignore it
-        unset($config['version_order']);
-        $this->assertAttributeEmpty('values', $config);
-        $this->assertAttributeEmpty('configFilePath', $config);
-        $this->assertNull($config->getConfigFilePath());
-    }
```

Comment:
These were removed because asserting private and protected properties is deprecated in phpunit. These tests don't really validate anything other than an empty constructor.